DuckDuckGo/AuthenticationViewController.swift

```diff
@@ -58,18 +58,37 @@ class AuthenticationViewController: UIViewController {
             onCouldNotAuthenticate()
         }
     }
+    
+    class Authenticator {
+        func canAuthenticate() -> Bool {
+            return true
```

Comment:
This is inaccurate; there are cases where authentication isn't available, so we can't return `true` at all times. It's up to the `LocalAuthentication` framework to tell us when it's available instead.